components/player/VideoPlayer.tsx

```diff
@@ -1,15 +1,65 @@
-import React from 'react';
-import { View } from 'react-native';
+/* eslint-disable react/no-unstable-nested-components */
+import { useTheme } from '@react-navigation/native';
+import { ResizeMode, Video } from 'expo-av';
+import { Image } from 'expo-image';
+import React, { useRef } from 'react';
+import { Dimensions, StyleSheet, View } from 'react-native';
+import { VIDEO_URL } from '../../config/constants';
 
-// TODO: Add VideoPlayer component
-/**
- * -  Implement a video player card component
- * - Hint: use `expo-av` to implement the video player with hidden controls
- * - consider adding fullscreen player
- */
+const { width } = Dimensions.get('window');
 
 const VideoPlayer = () => {
-  return <View />;
+  const { colors } = useTheme();
+
+  const video = useRef(null);
+
+  return (
+    <View style={[styles.container, { backgroundColor: colors.card }]} testID="video-player">
+      <Video
+        testID="video-component"
+        ref={video}
+        style={styles.video}
+        source={{
+          uri: VIDEO_URL,
+        }}
+        useNativeControls
+        resizeMode={ResizeMode.COVER}
+        isLooping
+        isMuted={false}
+        volume={0.3}
```

Comment:
That volume is too high on my phone by default and it is disturbing for the video to autoplay with sound on by default. The design has a custom mute icon on the top right of the video. We should start muted and hide the native controls and have the custom mute/unmute icon toggle the volume on and off.

![telegram-cloud-photo-size-4-5834776105012280515-y](https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/4d2b15ac-547f-46ba-a2c9-83cbcf64cbd3)

Reply:
it should be good to go now. please also test on physical android device

![Simulator Screen Shot - iPhone 14 pro - 2023-07-01 at 00 44 15](https://github.com/droidconKE/droidconKE2023ReactNative/assets/22290070/b28cae9e-4540-47d0-9b34-160026feccb5)